### PR TITLE
chore: add REUSE tool

### LINKED ISSUE #7: [rl-reuse_tool-2] Violation against OSS Rules of Play
A violation against the OSS Rules of Play has been detected.

Rule ID: rl-reuse_tool-2
Explanation: Does it have LICENSES directory with licenses? **No**

Find more information at: https://sap.github.io/fosstars-rating-core/oss_rules_of_play_rating.html

### LINKED ISSUE #9: [rl-reuse_tool-1] Violation against OSS Rules of Play
A violation against the OSS Rules of Play has been detected.

Rule ID: rl-reuse_tool-1
Explanation: Does README mention REUSE? **No**

Find more information at: https://sap.github.io/fosstars-rating-core/oss_rules_of_play_rating.html

### PR DESCRIPTION
**Pull request by SebastianWolf-SAP**
In line with SAP OSPO requirements, this PR integrates the REUSE tool

**Note:** Please register the repository with the REUSE Tool to complete the integration (see #10 and #11)

Fixes #7
Fixes #9

### CODE REVIEW

File: .reuse/dep5 [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: cloud-metrics-dashboard
```
Comment (yamladenov) [inline .reuse/dep5]: Upstream-Name should be "cloud-metricswatcher"

File: .reuse/dep5 [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ project’s code for that SAP External Product, subject to the terms of such
+ license agreement. If you do not have a valid license agreement for the use of
+ a particular SAP External Product, then you may only make use of any API Calls
+ in this project for that SAP External Product for your internal, non-productive
+ and non-commercial test and evaluation of such API Calls. Nothing herein grants
+ you any rights to use or access any SAP External Product, or provide any third
+ parties the right to use of access any SAP External Product, through API Calls.
+
+Files: *
+Copyright: 2015-2021 SAP SE or an SAP affiliate company and cloud-metrics-dashboard contributors
```
Comment (yamladenov) [inline .reuse/dep5]: "cloud-metrics-dashboard" should be "cloud-metricswatcher"

**Review by yamladenov — CHANGES_REQUESTED** [review]

**Comment by SebastianWolf-SAP** [comment]
Sorry, some copy/paste issues... I also added the missing copyright statement in the license file.

File: .reuse/dep5 [hunk 3]
```diff
@@ -0,0 +1,29 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: cloud-metricswatcher
+Upstream-Contact: Yavor Mladenov <yavor.mladenov@sap.com>
+Source: https://github.com/SAP/cloud-metrics-dashboard
```
Comment (yamladenov) [inline .reuse/dep5]: Sorry I just saw that the link is also pointing to another repository :)

File: LICENSES/Apache-2.0.txt [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+END OF TERMS AND CONDITIONS
+
+APPENDIX: How to apply the Apache License to your work.
+
+To apply the Apache License to your work, attach the following boilerplate notice, with the fields enclosed by brackets "[]" replaced with your own identifying information. (Don't include the brackets!)  The text should be enclosed in the appropriate comment syntax for the file format. We also recommend that a file or class name and description of purpose be included on the same "printed page" as the copyright notice for easier identification within third-party archives.
+
+Copyright [yyyy] [name of copyright owner]
```
Comment (yamladenov) [inline LICENSES/Apache-2.0.txt:61]: Should these placeholders also be changed

**Review by yamladenov — CHANGES_REQUESTED** [review]

File: .reuse/dep5 (hunk 3 above)
Comment (SebastianWolf-SAP) [reply .reuse/dep5]: :D Fixed

File: LICENSES/Apache-2.0.txt (hunk 4 above)
Comment (SebastianWolf-SAP) [reply LICENSES/Apache-2.0.txt:61]: As the copyright information is already in the dep5 file, this shouldn't be needed. Please also see https://reuse.software/faq/#license-templates However, in the end it's your choice - I explicitly enabled edits by maintainers, so feel free to change it if you want. However, please perform a `reuse lint` beforehand to ensure that the changes don't invalidate the REUSE tool integration.

Comment (yamladenov) [reply LICENSES/Apache-2.0.txt:61]: Ok thanks, we will see if we would do it

Comment (SebastianWolf-SAP) [reply LICENSES/Apache-2.0.txt:61]: Cool, thanks for merging. Please also ensure to have the project registered.

Comment (yamladenov) [reply LICENSES/Apache-2.0.txt:61]: yes I will see that should be done

**Pull request merged**